Designs/create_password/asset/style.scss

```
@@ -0,0 +1,29 @@
+.container{
+    
+    .card{
+        margin:0 auto;
+        max-width: 400px;
+        background-color: #b7b4b4;
+        padding:1rem;
+        h2{
+            text-align: center;
+            font-weight: 700;
+        }
+        p{
+            text-align: center;
+        }
+        .form-label{
+            font-weight: 400;
+        }
+
+        .reset-password{
+            .btn{
+                width: 100%;
+                border-radius: 29px;
```

Change border radius
```suggestion
                border-radius: 50px;
```